src/Persistence.Linnapps/ServiceDbContext.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace Linn.Products.Persistence.Linnapps
+{
+    using System.Collections.Generic;
+    using Domain.Linnapps.Products;
+
+    public class ServiceDbContext
+    {
+        public ServiceDbContext()
+        {
+            this.InitialiseTariffs();        
+        }
+
+        public List<Tariff> Tariffs { get; set; }
+
+        private void InitialiseTariffs()
+        {
```

Comment:
Why are these being set in here?

Reply:
it's either in here or in the repository.  When Oracle brings us the new driver (hopefully before Christmas) then this will all be refactored.